src/main/java/frc/robot/commands/IntakeCommand.java

```diff
@@ -0,0 +1,40 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.IndexerSubsystem;
+import frc.robot.subsystems.IntakeSubsystem;
+
+public class IntakeCommand extends Command {
+    private final IntakeSubsystem intakeSubsystem;
+
+    private final IndexerSubsystem indexerSubsystem;
+
+    /**
+     * Command to run the intake
+     * @param intakeSubsystem The instance of {@link IntakeSubsystem}
+     * @param indexerSubsystem The instance of {@link IndexerSubsystem} (needed for limebreak detection to stop intake motor)
+     */
+    public IntakeCommand(IntakeSubsystem intakeSubsystem, IndexerSubsystem indexerSubsystem) {
+        this.intakeSubsystem = intakeSubsystem;
+        this.indexerSubsystem = indexerSubsystem;
+
+        addRequirements(intakeSubsystem, indexerSubsystem);
+    }
+
+    @Override
+    public void execute() {
+        if (!indexerSubsystem.isCenter()) {
```

Comment:
How do you know the top or bottom of the note triggered this line break?

Do we need a PID here to keep the note in place? A power of 0 may let the note slip out?